packages/buidler-core/src/builtin-tasks/test.ts

```diff
@@ -20,7 +20,8 @@ export default function() {
     .addOptionalVariadicPositionalParam(
       "testFiles",
       "An optional list of files to test",
-      []
+      [],
+      types.array
```

Comment:
Removed `types.array` for now since my original intent would be backwards incompatible, and it's advantages are not clear considering the current param definitions implementation. commit ce41760bc04